### DISCUSSION TITLE
Setting multiple classes for the same modifier

### DISCUSSION THREAD

**Discussion by changethe** [tailwindlabs/tailwindcss#9135] [Ideas]
I'm not sure if this is already possible, but I couldn't find anything about it after searching for a while.

It would be nice to set multiple classes for the same modifier in one go... like `hover:[class1, class2, class3]`

It's not really a big deal to write out for 1-2 different classes for one modifier, but I sometimes find myself having to write out a single modifier 5 or more times on the same element. Would be really nice if multiple classes could be applied to the same modifier in a single block.

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

**Comment by changethe** [comment]
oh, thanks! i guess my searching skills suck (:
